examples/custom-prefetch-methods/custom_prefetch_methods.md

```diff
@@ -0,0 +1,22 @@
+# GraphQL service - Custom prefetch methods
+
+A prefetch method, in the context of Ballerina GraphQL is a method that is invoked before the actual resolver method. By default the prefetch method name has the convention of prefix `pre` followed by the resolver method name. If the user wants to use a custom prefetch method, the `prefetchMethodName` field of the `graphql:ResourceConfig` annotation can be used to override the default prefetch method name. This configuration is useful when the Query and Mutation operations have fields with the same name, and the user wants to use different prefetch methods for each field. To override the default prefetch method name, annotate the target field with the `graphql:ResourceConfig` annotation and specify the prefetch method name as the value of the `prefetchMethodName` field.
```

Comment:
```suggestion
A pre-fetch method in the context of Ballerina GraphQL is a method that is invoked before the actual resolver method. By default, the pre-fetch method name has the convention of having the `pre` prefix followed by the resolver method name. If you want to use a custom pre-fetch method, the `prefetchMethodName` field of the `graphql:ResourceConfig` annotation can be used to override the default pre-fetch method name. This configuration is useful when the `query` and `mutation` operations have fields with the same name and when it is required to use different prefetch methods for each field. To override the default pre-fetch method name, annotate the target field with the `graphql:ResourceConfig` annotation and specify the pre-fetch method name as the value of the `prefetchMethodName` field.
```
This paragraph is too long. Let's consider breaking it into multiple sentences.

Reply:
The std-lib BBE follow the convention of having a single paragraph for description. Are we enforcing any word limit for a paragraph ?

Reply:
Split into two paragraphs.